docs/developers/developers-setup.md

```diff
@@ -168,7 +168,7 @@ meson test
 To install Diffuse on your system (e.g. `/usr/local/`):
 
 ```sh
-meson install # requires admin privileges
+meson install # might require admin privileges
```

Comment:
It doesn't require admin priviledges all the time?

Reply:
Worked without, never asked for a password.

Reply:
Would be nice to know if it never requires it, or if it's just because you have special settings on your system, but for now, it'll do just fine.